WordPress/Classes/System/RootViewCoordinator.swift

```diff
@@ -64,7 +64,7 @@ class RootViewCoordinator {
     // MARK: JP Features State
 
     /// Used to determine if the Jetpack features are enabled based on the removal phase.
-    private static func shouldEnableJetpackFeatures(featureFlagStore: RemoteFeatureFlagStore) -> Bool {
+    private static func shouldShowJetpackFeaturesBasedOnCurrentPhase(featureFlagStore: RemoteFeatureFlagStore) -> Bool {
```

Comment:
[nit] Super tiny nit but I think we should change the comment too, from "enable" to "show"

Reply:
Great catch! Updated in bff22377dbbea9435f77fdf2a4bfa9b83ab78624 👍